app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,16 @@
+<% unless current_page?(new_user_session_path)%>
+<nav class="flex justify-end w-full">
+   <div class="flex p-5 justify-between w-2/3 px-20">
+   <ul class="flex justify-between w-3/5">
+   <li><%= link_to  'Food Ingredents', user_foods_path(current_user)%></li>
```

Comment:
- [ ] [REQUIRED] Great project @tylher and @wuletawwonte 👍🏽 , However, you need to take a look at the registration with devise as I can not signup currently. Looks like your navbar is hindering the signup. Kindly fix this.
![Screenshot from 2022-08-06 00-00-41](https://user-images.githubusercontent.com/58692590/183222184-6e2a27cf-1267-491a-8b4f-27cd8d0ea4ef.png)

Reply:
Thank you @Whoistolu, I understand this problem and we will make the corrections as soon as possible. 👍